jekyll/_cci2/next-gen-migration-guide.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: classic-docs
+title: "Migrating to next-gen Convenience Images"
+short-title: "Migrating to next-gen Convenience Images"
+description: "Migrating to next-gen Convenience Images"
+order: 30
+version:
+- Cloud
+- Server v2.x
+---
+
+* TOC
+{:toc}
+
+
+## Overview
+
+Last year (2020) CircleCI introduced the next generation (next-gen) of Convenience Images. These new images are designed to replace the legacy Convenience Images that came out during the announcement of CircleCI 2.0, about 4 years ago. The next-gen CircleCI Convenience Images are designed from the ground up for a Continuous Integration environment. They’re designed to be faster, more efficient, and more importantly, more reliable. You can learn more about all of the features on our blog post. As we begin to deprecate the legacy images, here’s information on the migration process.
+
+Moving from a legacy to next-gen image is easy and straightforward. All legacy images have a Docker namespace of “circleci” while next-gen images have a Docker namespace of “cimg”. For example, migrating from the legacy Ruby or Python image to the respective next-gen:
+
+circleci/ruby:2.3.0 -> cimg/ruby:2.3.0
+circleci/python:3.8.4 -> cimg/python:3.8.4
+
+
+## Changes
+
+### Deprecated images
+
+There are a few images that will be completely deprecated, without a replacement, as well as one that has a name change. The buildpack-deps, JRuby, and DynamoDB images will be fully deprecated with no next-gen equivalents. If you are using the buildpack-deps image, the suggestion is to use the new CircleCI Base image, cimg/base. For the other two images, you can install the software yourself in the base image or use a 3rd-party image instead. The Go image name has been changed from golang to go. All of the Legacy to Next-Gen image changes are captured below in this table:
+
+
+| Legacy Image | Next-Gen Image |
+| --- | --- |
+| circleci/buildpack-deps | cimg/base |
+| circleci/jruby | No suggested path |
+| circleci/dynamodb | No suggested path |
+| circleci/golang | cimg/go |
+
+
+### Browser Testing
+
+With legacy images, there were 4 different variant tags you could use to get browser testing for a particular image. For example, if you were doing browser testing with the Python v3.7.0 image, you might have used Docker image: circleci/python:3.7.0-browsers. These 4 tags have now been consolidated into a single tag designed to be used together with the CircleCI Browser Tools orb.
+
+
+Legacy Variant Tags
+
+
+Next-gen Variant Tag
+-browsers
+-browsers + Browser Tools Orb
+-browsers-legacy
+-node-browsers
+-node-browsers-legacy
```

Comment:
Not sure if I'm misunderstanding, but are these mixed up? Should there be something under the "Legacy Variant Tags" section? Should these be list-items? If so, should -browsers be `-browsers` with ?

Reply:
Yes I am wondering the same?

Reply:
Yeah this is messed up. It was a table that lost its formatting when the guide was converted over. I am fixing it now and looking for other places where these conversion problems might have occured.